src/block-management/inline-toolbar/index.js

```diff
@@ -52,19 +54,42 @@ export default class InlineToolbar extends React.Component<PropsType> {
 	}
 
 	render() {
+		const { order } = this.props;
+		const moveUpButtonTitle = this.props.canMoveUp ? sprintf(
+			/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
+			__( 'Move up from row %1$s to row %2$s' ),
+			order,
+			order - 1 ) : sprintf(
+			/* translators: accessibility text. %d: block position (number) */
+			__( 'Move up from row %1$s' ),
```

Comment:
It would be great if we can say here just `Move up`, for this case where the block can't be moved up.
The system will add a `dimmed | disabled` to let the user know that the action is disabled. And the user will knowledge the intent of the button.

Reply:
I think both messages are ok but, I agree removing the row number in this case this lines up better states the intent of the button when disabled and can be better for accessibility. As it relates to the tests this shouldn't have any impact so 👍